.travis.yml

```diff
@@ -1,18 +1,18 @@
 language: php
-php:
-  - 7.1
-  - 7.2
-  - nightly
+sudo: false
```

Comment:
Isn't `sudo: false` redundant under Trusty builds?

Reply:
The reference doc doesn't list what the default is.  Personally, I lean towards always specifying to avoid ambiguity.
https://docs.travis-ci.com/user/reference/trusty/

This other doc does not explicitly say that container is default; but lists cases that do force Sudo enabled VM (assuming there is no conflict)
https://docs.travis-ci.com/user/reference/overview/#For-a-particular-.travis.yml-configuration